app/controllers/services_controller.rb

```diff
@@ -14,6 +14,7 @@ def edit
 
   def update
     if @bibliography_service.update(update_params)
+      @exhibit.sync_bibliography if @bibliography_service.api_settings_changed?
```

Comment:
It seems like something with such dramatic side-effects like this in an ActiveRecord callback could be problematic.

This would make it impossible to update the model without syncing the service though, right? (without explicitly preventing callbacks, that is)